### PR TITLE
[Update] LKE collection property `version` to k8s_version

### PR DESCRIPTION
**Pull request by leslitagordita**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -15756,8 +15756,7 @@ components:
           x-linode-filterable: true
           x-linode-cli-display: 3
           example: us-central
-        version:
-          x-linode-filterable: true
```
Comment (adammohammed) [inline openapi.yaml:15760]: This should be handled in #237

**Comment by leslitagordita** [comment]
Yes! Just pointed it at the feature branch and was thinking the same thing. Your PR https://github.com/linode/linode-api-docs/pull/237 will handle the removal of the other filterable properties

**Comment by leslitagordita** [comment]
CC: @asauber, @Dorthu <- for release alignment purposes

**Comment by asauber** [comment]
Thank you @leslitagordita. Yes, we will want to do `linode-cli` release, as soon as the current hotfix is in prod.

**Pull request merged**